### DISCUSSION TITLE
How to open app in background message receiving ?

### DISCUSSION THREAD

**Discussion by Faiizii** [firebase/flutterfire#5774] [Q&A]
````
Future handleBackgroundMessage(RemoteMessage message) async {
  print(message.data);//works perfectly.
}
````
I want to lunch the app or start a screen on receiving new push notification. but flutter says not able find the context. 
So any idea how do we do this in background message handler ? 
Thanks in advance.

**Comment by alexandrefett** [comment]
Did you find a solution? I have the same problem here
I tried local_notification but no success when the app is closed

**Reply by Faiizii** [reply]
unfortunately not yet @alexandrefett , I think we have to handle it in native code sadly.

**Reply by arefrafei92** [reply]
can you help me how we can handle it in native ? I want to save notification in database

**Reply by vanshikabatman** [reply]
Got any solution?
